lua/weapons/weapon_lfsmissilelauncher.lua

```diff
@@ -146,7 +150,7 @@ function SWEP:Think()
 			local stuff = WorldToLocal( vehicle:GetPos(), Angle( 0, 0, 0 ), startpos, Owner:EyeAngles() + Angle(90,0,0) )
 			local dist = stuff:Length()
 
-			if dist < ClosestDist and Ang < SmallestAng then
+			if dist < ClosestDist and Ang < SmallestAng then -- only switch when much closer!

```

Comment:
nit: prefer comments above lines

```suggestion
			-- only switch when much closer!
			if dist < ClosestDist and Ang < SmallestAng then
```

Reply:
Applied.